parlai/core/dict.py

```diff
@@ -129,6 +136,12 @@ def add_cmdline_args(argparser):
             dictionary.add_argument(
                 '--dict-lower', default=DictionaryAgent.default_lower, type='bool',
                 help='Whether or not to lowercase all text seen.')
+            dictionary.add_argument(
```

Comment:
how do you turn off bpe?

Reply:
BPE is only enabled if you pass `--dict-tokenizer bpe` as an argument.